WalletWasabi.Fluent/Models/Wallets/WalletModel.cs

```diff
@@ -70,7 +70,7 @@ public WalletModel(Wallet wallet, IAmountProvider amountProvider)
 
 	public IAddressesModel Addresses { get; }
 
-	internal Wallet Wallet { get; }
+	public Wallet Wallet { get; }
```

Comment:
With the current design, it doesn't have to be public. You can revert.

Reply:
Done